src/dynatrace/opentelemetry/metrics/export/__init__.py

```diff
@@ -12,35 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import logging
-import math
-import requests
 from typing import Mapping, Optional
 
 import opentelemetry.sdk.metrics as metrics
-
-from dynatrace.metric.utils import (
-    DynatraceMetricsSerializer,
-    DynatraceMetricsApiConstants,
-    DynatraceMetricsFactory,
-    MetricError
-)
-
 from opentelemetry.sdk.metrics.export import (
-    MetricExporter,
-    MetricExportResult,
-    Sum,
     AggregationTemporality,
-    Gauge,
-    Histogram,
-    MetricsData,
-    DataPointT,
-    NumberDataPoint,
-    Metric,
-    HistogramDataPoint,
     PeriodicExportingMetricReader
 )
 
+from dynatrace.opentelemetry.metrics.export._exporter import (
+    _DynatraceMetricsExporter
+)
+
 VERSION = "0.3.0-rc1"
 
 _DYNATRACE_TEMPORALITY_PREFERENCE = {
```

Comment:
Now there is no way for users to set this anymore, right? If that is the case, we can just keep it private.

Reply:
Isn't it already private? :thinking:

Reply:
Yup! I just wanted to make sure that I understood correctly that this cannot be set anymore when using the DynatraceExporter.